### ISSUE TITLE
Build landing page

### ISSUE THREAD

**Issue by abertschi** [abertschi/water-me#9]

**Comment by inigochoa** [comment]
Hey!
do you want help building a landing page to promote the app?

**Comment by abertschi** [comment]
Hey back,

Yes - do you want to work on this?

**Comment by inigochoa** [comment]
Sure. Do you have any design in mind?

**Comment by abertschi** [comment]
No, not really. It would be nice if the site reflects the colors of the app itself, but does not have to. It can resemble a "conventional" app landing which informs of the feature and shows some screenshots.

For instance:
https://dribbble.com/search/landing-page

But feel free to bring in your own ideas. If you want feedback feel free to post some design iterations here.

It should further be mobile friendly and can link to the github for code and fdroid for binary.

cheers

**Comment by inigochoa** [comment]
Ok. Will check some ideas and keep you up to date

**Comment by inigochoa** [comment]
Hey! I've been working on the desktop version. Let me know what you think

![Screenshot 2023-08-23 at 21-55-04 Water Me 🪴](https://github.com/abertschi/water-me/assets/7983569/58bb8081-0bed-4db6-8990-08b5bd1f5c71)

**Comment by abertschi** [comment]
Great, Looks good. We just have to make sure it is also mobile friendly and renders well on a phone.

**Comment by inigochoa** [comment]
Sure. I've been working on them this morning. I also modified some sizes to make spacing better.

Desktop:

![Screenshot 2023-08-24 at 13-23-38 Water Me 🪴](https://github.com/abertschi/water-me/assets/7983569/aa037b7f-4f88-439c-9545-ea429b15d540)

Tablet:

![Screenshot 2023-08-24 at 13-23-30 Water Me 🪴](https://github.com/abertschi/water-me/assets/7983569/163b5369-832e-4cdd-a4dd-726b547209c9)

Mobile:

![Screenshot 2023-08-24 at 13-23-17 Water Me 🪴](https://github.com/abertschi/water-me/assets/7983569/c0c0856c-aa20-4f91-a2fd-85096dbde645)

If you want to change anything, just let me know. Otherwise, we should be thinking about the content.

**Comment by inigochoa** [comment]
Hey @abertschi !

I deployed the site to [GitHub pages](https://inigochoa.github.io/water-me-landing/) so you can check it on your own. Keep in mind that it lacks content and animations.

Let me know what you think

**Comment by inigochoa** [comment]
Sure. Just wanted to place something there. Maybe a logo with the app name on the right? The app name in a fancy font?

If you like the rest, we should talk about the content of each section

**Comment by abertschi** [comment]
> Sure. Just wanted to place something there. Maybe a logo with the app name on the right? The app name in a fancy font?

Sounds good. Yeah i dont have something in mind, you can be creative :)

As for the content: we can use the content from the README, without the technical details. The purpose of the app should be quite clear and the functionalities are not complicated. Either you write some things on your own and I change it later on or I post some ideas in another comment.

**Comment by abertschi** [comment]
Some ideas:

- Keep your plants happy and healthy.

- Water me is a simple plant watering reminder app. It's simple, open source, and it respects your privacy. No ads, no tracking, no accounts – just pure plant care!

- Customizable Watering Schedule
- Create Pictures, add Plant Details
- Notification Reminders
- No Remote Entity
- Import and Export
- Free and Open-Source
- Runs on Android

**Comment by inigochoa** [comment]
Hey!

I made some updates to the landing site. Let me know what do you think.

**Comment by inigochoa** [comment]
Hey @abertschi !

Hope everything is OK

**Comment by abertschi** [comment]
![image](https://github.com/abertschi/water-me/assets/2311941/679d228d-4d66-4698-b8b2-75b5a69b3967)

The the layout looks scarce on desktop, if you can come up with more text feel free to do so, otherwise I suggest you create a pull request to merge the source code into ./landing of the root of this repository so we get this merged into master.  Also, add yourself to the authors file :+1:

**Comment by abertschi** [comment]
Hi @inigochoa, thank you for all your work.
I suggest we merge it into /web/ of this repository. Do you want me to copy it myself or will you create a PR? thanks+cheers.
